index.html

```diff
@@ -1416,6 +1416,15 @@ <h3>Types</h3>
               </td>
             </tr>
 
+            <tr>
+              <td>
+<a href="#confirmation-method">confirmationMethod</a>&nbsp;object
+              </td>
+              <td>
+A valid confirmation method <a>type</a>. For example,<br>
+<code>"type": "VerificationKeyConfirmation2023"</code>
```

Comment:
We will need at least one real example.

Reply:
Real example is in the PR. There are even two.